Packs/Netskope/Integrations/NetskopeAPIv1/README.md

```diff
@@ -0,0 +1,754 @@
+| File.EntryID | String | The entry ID of the file. |
+| File.Info | String | File information. |
+| File.Type | String | The file type. |
+| File.Extension | String | The file extension. |
+
+#### Command example
+```!netskope-quarantined-file-get file_id=1M_RU4jLPUwclKOhqZ7sPSqkMNS-S6Vyr quarantine_profile_id=1```
+#### Context Example
+```json
+{
+    "File": {
+        "EntryID": "4447@8479e914-8493-4968-8f32-78852375d17b",
+        "Extension": "zip",
+        "Info": "application/zip",
+        "MD5": "8ac692ef2cc78adfc523188e54d52933",
+        "Name": "1M_RU4jLPUwclKOhqZ7sPSqkMNS-S6Vyr.zip",
+        "SHA1": "19cf128ad8fd41108d0a601e8e9c8cf123c11c5a",
+        "SHA256": "9ea98de9c4f852665f678338767280f64b20ed11943c49709440643ed04df122",
+        "SHA512": "5a6bce0fd15e4184d29ee8d38627ec47150665a6ccb67d39b02a8898ff0c987c4e7be580c9243cc3d4a7f3f2b637f226216434bf01bc2babfe35a4b7ff8cc2cc",
+        "SSDeep": "384:wc313ff+IizBt/oPervm6X61XWRcjDQJB9k3Wl:wMGIct/PjtX61EVB9mWl",
+        "Size": 15774,
+        "Type": "Microsoft Excel 2007+"
+    }
+}
+```
+
+#### Human Readable Output
+
+
+
+### netskope-quarantined-file-update
+***
+Take an action on a quarantined file.
+
+
+#### Base Command
+
+`netskope-quarantined-file-update`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| quarantine_profile_id | The profile ID of the quarantined file. | Required |
+| file_id | The ID of the quarantined file. | Required |
+| action | Action to be performed on a quarantined. Possible values are: block, allow. | Required |
+
+
+#### Context Output
+
+There is no context output for this command.
+
+### Command example
+!netskope-quarantined-file-update file_id=1M_RR4jLPUwclKOhqZ7sPSqkMNS-S6Vyr quarantine_profile_id=1 action=block
+
+#### Human Readable Output
+
+>The file 1M_RR4jLPUwclKOhqZ7sPSqkMNS-S6Vyr was successfully blocked!
+
+
+### netskope-url-list-update
+***
+Update the URL List with the values provided.
+
+
+#### Base Command
+
+`netskope-url-list-update`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| name | Name of an existing URL List shown in the Netskope UI on the URL List page. | Required |
+| urls | The content of the URL list. | Required |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Netskope.URLList.name | String | The name of the URL list. |
+| Netskope.URLList.URL | String | The content the URL list. |
+
+#### Command example
+```!netskope-url-list-update name="Allowed URLs" urls="allow.me,allow2.me"```
+#### Context Example
+```json
+{
+    "Netskope": {
+        "URLList": {
+            "URL": [
+                "allow.me",
+                "allow2.me"
+            ],
+            "name": "Allowed URLs"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>URL List Allowed URLs:
+>allow.me, allow2.me
+
+### netskope-file-hash-list-update
+***
+Update file hash list with the values provided.
```

Comment:
@RazOmry We need to state that the update action will rewrite the existing values in the list and doesn't add to them.